Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -2486,8 +2520,14 @@ def fetch_incidents(client, first_fetch_time, integration_instance, last_run: di
     if incidents_from_previous_run:
         raw_incidents = incidents_from_previous_run
     else:
-        raw_incidents = client.get_incidents(gte_creation_time_milliseconds=last_fetch,
-                                             limit=max_fetch, sort_by_creation_time='asc')
+        if statuses:
+            raw_incidents = []
+            for status in statuses:
+                raw_incidents += client.get_incidents(gte_creation_time_milliseconds=last_fetch, status=status,
+                                                      limit=max_fetch, sort_by_creation_time='asc')
```

Comment:
one last thing that just came to my mind:

the fetch_incidents counts on the `raw_incidents` variable to be sorted by the creation time of the incidents, but because you now fetch in batches by status, this could take the variable out of order. We need to add a resorting after all the incidents are gathered.

```raw_incidents = sorted(raw_incidents, key = lambda inc: inc['creation_time'])```